{"name": "test_auth_user", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'cookies'", "trace": "self = <tests.test_auth.TestUserAuth object at 0x101d44370>\n\n    def setup(self):\n        data = {\n            'email': 'vinkotov@example.com',\n            'password': '1234'\n        }\n    \n        response1 = MyRequests.post(\n            \"/user/login\",\n            data=data\n        )\n    \n>       self.auth_sid = self.get_cookie(response1,\"auth_sid\")\n\ntests/test_auth.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.test_auth.TestUserAuth object at 0x101d44370>, response = None, cookie_name = 'auth_sid'\n\n    def get_cookie (self, response: Response, cookie_name):\n>      assert cookie_name in response.cookies, f\"Куки был не найден с именем {cookie_name} в последнем ответе\"\nE      AttributeError: 'NoneType' object has no attribute 'cookies'\n\nlib/base_case.py:6: AttributeError"}, "description": "This test successfully authorize user by email and password", "start": 1682237411153, "stop": 1682237411153, "uuid": "c59e37d4-df64-45fe-9414-5506e4f46282", "historyId": "dadb9f82538e912234592dc4e33acc8c", "testCaseId": "ee0d3c284ef17e3bfe9f7dd18d65d110", "fullName": "tests.test_auth.TestUserAuth#test_auth_user", "labels": [{"name": "epic", "value": "Authorization cases"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "subSuite", "value": "TestUserAuth"}, {"name": "host", "value": "MacBook-Pro-Renat.local"}, {"name": "thread", "value": "10082-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}